Tests/Editor/Actions/StripProBuilderScripts.cs

```diff
@@ -0,0 +1,27 @@
+﻿using UObject = UnityEngine.Object;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.ProBuilder;
+using UnityEngine.ProBuilder.MeshOperations;
+
+public class StripProBuilderScripts
+{
+    [Test]
+    public void StripProBuilderScripts_RemovesMeshAndPolyShapeComponents()
+    {
+        var go = new GameObject();
+        var poly = go.AddComponent<PolyShape>();
+        go.AddComponent<ProBuilderMesh>();
+        poly.CreateShapeFromPolygon();
+
+        Assume.That(go.GetComponent<ProBuilderMesh>() != null);
+        Assume.That(go.GetComponent<PolyShape>() != null);
+
+        UnityEditor.ProBuilder.Actions.StripProBuilderScripts.DoStrip(go.GetComponent<ProBuilderMesh>());
```

Comment:
Add a `using UnityEditor.ProBuilder.Actions` instead of writing it out for more readable code.

Reply:
I used this because I names both classes the same way (oups), so renamed the Test class